crates/lib/test/fixtures/rules/std_rule_cases/CV09.yml

```diff
@@ -0,0 +1,247 @@
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_regex: ^.*(2022_06_01|2022_05_01).*$
+
+test_fail_bigquery:
+  fail_str: |
+    SELECT *
+    FROM `owner.schema.table_2022_06_01_desktop`
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_regex: ^.*(2022_06_01|2022_05_01).*$
+
+test_fail_bigquery2:
+  fail_str: |
+    SELECT *
+    FROM `owner.schema.table_2022_06_01_desktop`
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_regex: .*(2022_06_01|2022_05_01).*
+
+test_fail_bigquery3:
+  fail_str: |
+    SELECT *
+    FROM `owner.schema.table_2022_06_01_desktop`
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_regex: (2022_06_01|2022_05_01)
+
+test_pass_comment_word1:
+  pass_str: |
+    SELECT *
+    FROM table1
+    -- TABLESAMPLE SYSTEM (.05 PERCENT)
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_words: TABLESAMPLE
+
+test_pass_comment_word2:
+  pass_str: |
+    SELECT *
+    FROM table1
+    # TABLESAMPLE SYSTEM (.05 PERCENT)
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_words: TABLESAMPLE
+
+test_pass_comment_word3:
+  pass_str: |
+    SELECT *
+    FROM table1
+    /*
+    TABLESAMPLE SYSTEM (.05 PERCENT)
+    */
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_words: TABLESAMPLE
+
+test_pass_comment_regex1:
+  pass_str: |
+    SELECT *
+    FROM table1
+    -- TABLESAMPLE SYSTEM (.05 PERCENT)
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_regex: (TABLESAMPLE)
+
+test_pass_comment_regex2:
+  pass_str: |
+    SELECT *
+    FROM table1
+    # TABLESAMPLE SYSTEM (.05 PERCENT)
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_regex: (TABLESAMPLE)
+
+test_pass_comment_regex3:
+  pass_str: |
+    SELECT *
+    FROM table1
+    /*
+    TABLESAMPLE SYSTEM (.05 PERCENT)
+    */
+  configs:
+    core:
+      dialect: bigquery
+    rules:
+      convention.blocked_words:
+        blocked_regex: (TABLESAMPLE)
+
+test_pass_match_source1:
+  pass_str: |
+    SELECT * FROM {{ ref('deprecated_table') }}
+  configs:
+    core:
+      templater: jinja
+      ignore_templated_areas: true
+    rules:
+      convention.blocked_words:
+        blocked_regex: ref\('deprecated_
+        match_source: true
+
+test_pass_match_source2:
+  pass_str: |
+    SELECT * FROM {{ ref('deprecated_table') }}
+  configs:
+    core:
+      templater: jinja
+      ignore_templated_areas: false
+    rules:
+      convention.blocked_words:
+        blocked_regex: ref\('deprecated_
+        match_source: false
+
+test_fail_match_source1:
+  fail_str: |
```

Comment:
```suggestion
test_fail_match_source1:
  ignored: "jinja is not supported"
  fail_str: |
```

Reply:
I did this slightly differently; let me know what you think.